SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/LayeredLBStrategyTest.cpp

```diff
@@ -0,0 +1,412 @@
+#include "LoadBalancing/LayeredLBStrategy.h"
```

Comment:
Can you put a copyright comment on the first line of the file? Should be copy-able from other files

Reply:
done